lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +17,19 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str) or not text:
+        return None
+    punctuation = """.,;:¡!¿?…⋯‹›«»/\\"“”[]()⟨⟩}{&|-–~—"""
+    for i in text:
+        if i in punctuation:
+            text = text.replace(i, '.')
```

Comment:
remove `.` from `punctuation` if you use `.` as a special character

Reply:
removed